### PR TITLE
Fix Hindi translation and translate more strings

### PR DESCRIPTION
**Pull request by virejdasani**
# Description

There are typos and errors in the opsdroid/locale/hi/LC_MESSAGES/opsdroid.po
For example, the quotations are applied improperly. This causes some of the text to leak out of the string.

I have fixed all of these issues in my pull request by going line by line and fixing typos and quotation errors

Fixes #<1646>

## Status
**READY**

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

# Checklist:

- [ x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by virejdasani** [comment]
I have also added more translations that were previously commented out

File: opsdroid/locale/hi/LC_MESSAGES/opsdroid.po [hunk 1]
```diff
@@ -303,11 +303,10 @@ msgid "Started opsdroid %s"
 msgstr "opsdroid %s शुरू"
 
 #: opsdroid/matchers.py:127
-msgid ""
-"Recast.AI is now called SAP Conversational AI, this matcher  will stop "
```
Comment (FabioRosado) [inline opsdroid/locale/hi/LC_MESSAGES/opsdroid.po:307]: This line needs to stay the same, that's why the tests are failing. Our localization expects the strings to be literal ones from the code base. 

The previous PR that added the Hindi translation, probably followed the steps for translating opsdroid.

**Review by FabioRosado — CHANGES_REQUESTED** [review]

**Comment by FabioRosado** [comment]
Hello @virejdasani  thank you for Raising this PR and wanting to help the project. 

I've added a comment on the translation strings - they need to be set as they were otherwise the tests will fail - like they are now.

You can check the steps for translating to a language on the [localization documentation](https://docs.opsdroid.dev/en/stable/contributing.html?highlight=translation#localization)

**Review by FabioRosado — APPROVED** [review]
Hello @virejdasani  thank you for updating this PR. I've noticed that you commented out some of the translations that you added previously, is this intentional? could be good to have them in the file.

I'm setting this PR as approved so it counts for your Hacktoberfest, but would be great to hear from you about this 👍

**Comment by virejdasani** [comment]
Oh, I'm sorry, I did'nt realize I did that. I'll get to that ASAP

**Comment by FabioRosado** [comment]
Awesome thanks! As soon as CI finishes I will merge this, thank you so much for raising the PR and for spotting and fixing the issue 😄 👍

**Comment by virejdasani** [comment]
I am extremely sorry, I am not sure why the tests failed. 
This is my first time doing this. Can you help me out?

**Comment by FabioRosado** [comment]
Again thank you so much for fixing those annoying errors and for the patience this PR is now merged 😄 👍

**Comment by virejdasani** [comment]
Thanks a lot!
Also, I just wanted to ask, do you know why the PR is still not accepted in my Hacktoberfest?
I assumed it would be accepted as soon as this merge happens...

**Comment by FabioRosado** [comment]
With the new changes they said that any PR that has been approved, merged or has the hacktoberfest-accepted tag will count so It should be added straight away unless they take a while to process it now?

I'm going to work now, if in two hours it doesn't slow let me know and I'll add the "hacktoberfest-approved" tag.

**Comment by virejdasani** [comment]
It has been over 2 hours ad the PR is still not accepted.
I'd appreciate it if you could tag it with the hacktoberfest tag.
Thanks in advance!

**Comment by virejdasani** [comment]
One day later, they have still not accepted it...

**Comment by FabioRosado** [comment]
I've added the hacktoberfest-accepted label as well. This shouldn't be needed since it was approved and merged but perhaps the label will unlock the PR in your hacktoberfest dashboard

**Comment by virejdasani** [comment]
I'm not sure why, but for some reason, the PR is still in review.
hacktober-in-review
Anyway, thanks for adding the tag

**Comment by virejdasani** [comment]
Thanks a lot!

**Pull request merged**